Minutes.java

```diff
@@ -1,13 +1,28 @@
 class Minutes extends ConsoleProgram {
 
   /**
-  * Description
-  * @author:
+  * This code lets you enter a number of hours and convert it into days and hours.
+  * @author: A. Emmanuel-Singh
   */
   
   public void run() {
     
-    // start coding here
-    
+    // Declare Variables 
+    int intMin;
+    int intHours;
+    int intDays;
+    int intOutputMin;
+
+    // Input from User
+    intMin = readInt("Enter the number of minutes you would like converted into Days, Hours, and Minutes: ");
+
+    // Calculations 
```

Comment:
This code works, it output the same answer, you can change the code to: 
intDays = intMinutes / 1440;
intOutputMin = intMinutes % 1440;